src/krate.rs

```diff
@@ -598,6 +602,32 @@ impl Crate {
         Ok(users.chain(teams).collect())
     }
 
+    pub fn owner_team(&self, conn: &PgConnection) -> CargoResult<Vec<Owner>> {
+        let base_query = CrateOwner::belonging_to(self).filter(crate_owners::deleted.eq(false));
+        let teams = base_query
+            .inner_join(teams::table)
+            .select(teams::all_columns)
+            .filter(crate_owners::owner_kind.eq(OwnerKind::Team as i32))
+            .load(conn)?
+            .into_iter()
+            .map(Owner::Team);
+
+        Ok(teams.collect())
+    }
+
+    pub fn owner_user(&self, conn: &PgConnection) -> CargoResult<Vec<Owner>> {
```

Comment:
`Crate` has a lot of behavior on it as is. What do you think about putting this on `User` instead and calling it `User::owning(crate: &Crate, conn: &PgConnection)`